frontend/lib/src/components/widgets/FileUploader/FileDropzone.test.tsx

```diff
@@ -32,26 +33,28 @@ const getProps = (props: Partial<Props> = {}): Props => ({
 describe("FileDropzone widget", () => {
   it("renders without crashing", () => {
     const props = getProps()
-    const wrapper = shallow(<FileDropzone {...props} />)
+    render(<FileDropzone {...props} />)
 
-    expect(wrapper).toBeDefined()
+    expect(screen.getByTestId("stDropzone")).toBeInTheDocument()
   })
 
   it("renders dropzone without extensions", () => {
     const props = getProps({
       acceptedExtensions: [],
     })
-    const wrapper = shallow(<FileDropzone {...props} />)
-    const dropzoneWrapper = wrapper.find(Dropzone)
-    expect(dropzoneWrapper.props().accept).toBe(undefined)
+    render(<FileDropzone {...props} />)
+    expect(screen.queryByTestId("stDropzoneInput")).not.toHaveAttribute(
+      "accept"
+    )
   })
 
   it("renders dropzone with extensions", () => {
     const props = getProps({
       acceptedExtensions: [".jpg"],
     })
-    const wrapper = shallow(<FileDropzone {...props} />)
-    const dropzoneWrapper = wrapper.find(Dropzone)
-    expect(dropzoneWrapper.props().accept).toEqual([".jpg"])
+    render(<FileDropzone {...props} />)
+    expect(screen.getByTestId("stDropzoneInput").getAttribute("accept")).toBe(
```

Comment:
Also this one - `expect(screen.queryByTestId("stDropzoneInput")).toHaveAttribute("accept", ".jpg")`